### ISSUE TITLE
Without feature distillation, it can get even better results?

### ISSUE THREAD

**Issue by tanglw2020** [ArchipLab-LinfengZhang/pytorch-self-distillation-final#3]
Training resnet18 in cifar100
use the original code and settings, the best result is
Test Set AccuracyAcc: 4/4: 81.3300% 3/4: 81.0600% 2/4: 79.3000%  1/4: 76.2500% Ensemble: 82.3200%

Without feature distillation,  
the best result is
Test Set AccuracyAcc: 4/4: 81.7100% 3/4: 81.3000% 2/4: 78.1800%  1/4: 75.2800% Ensemble: 82.5700%
              
Is the phenomenon just a coincidence? My machine could not do more complex experiments to check it.

**Comment by hello451** [comment]
I have got similar situation... So maybe the most important thing is the classifiers from shallow to deep

**Comment by deepzlk** [comment]
yes,i have the similar result.
